src/path.ts

```diff
@@ -212,9 +213,14 @@ export const extname: typeof path.extname = function (p) {
 
 // relative
 export const relative: typeof path.relative = function (from, to) {
-  const _from = resolve(from).replace(_ROOT_FOLDER_RE, "$1").split("/");
-  const _to = resolve(to).replace(_ROOT_FOLDER_RE, "$1").split("/");
-
+  const _from = resolve(from)
```

Comment:
Might we apply driver letter to upper-case normalization also directly from `resolve` and `normalize` (`normalizeWindowsPath`) for better consistency?

Reply:
I'm very happy with that. Only, it's a bigger change.